tests/Unit/Application/Regulation/Command/Steps/Factory/SaveRegulationStep4CommandFactoryTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Domain\Regulation\Command\Steps\Factory;
+
+use App\Application\Regulation\Command\Steps\Factory\SaveRegulationStep4CommandFactory;
+use App\Domain\Condition\VehicleCharacteristics;
+use App\Domain\Regulation\RegulationOrderRecord;
+use PHPUnit\Framework\TestCase;
+
+final class SaveRegulationStep4CommandHandlerTest extends TestCase
```

Comment:
```suggestion
final class SaveRegulationStep4CommandFactoryTest extends TestCase
```

Si on applique ma suggestion de passer ça dans la commande directement, on aurait alors un `SaveRegulationStep4CommandTest.php`